README.md

```diff
@@ -32,15 +32,14 @@ Once the cloud resources are provisioned, see the [viya4-deployment](https://git
 This project supports running **either** via terraform installed on your local machine **or** via a docker container. 
 
 - Access to an **Azure Subscription** and [**Identity**](./docs/user/TerraformAzureAuthentication.md) with '*Contributor*' role
-  - #### Terraform
+  - #### Terraform (4 dependencies)
```

Comment:
Why were the extra items added here to both the Terraform and Docker sections? The list should be sufficient. The items listed are dependencies and are needed.

Reply:
The items I am asking about are `(4 dependencies)`

Reply:
Remove the text: (4 dependencies)

Reply:
@andybouts when will this be addressed?

Reply:
It has been removed, must have been an oversight.  That said, I think this would be more clear to the end users if it said "Terraform Dependencies" and "Docker Dependencies" or "Terraform Prerequisites" and "Docker Prerequisites".